### PR TITLE
AUT-259 configuration endpoints_part1

### PR DESCRIPTION
**Pull request by azayatsfv**
GET get_configuration_list
GET get_fees
GET get_loan_pipeline

### CODE REVIEW

File: APIs/client.py [hunk 1]
```diff
@@ -29,6 +30,7 @@ def __init__(self, base_url, database, port=None, headers=None):
         self.assets = AssetsClient(client=self)
         self.borrowers = BorrowersClient(client=self)
         self.companies = CompaniesClient(client=self)
+        self.configuration = ConfigurationClient(client=self)
```
Comment (darkhipenko) [inline APIs/client.py:33]: `configuration ` or `los_configuration` - globally

**Review by darkhipenko — CHANGES_REQUESTED** [review]
Naming of LOS Configuration should be consistent everywhere. 
Please, refer to the comment.

File: APIs/client.py (hunk 1 above)
Comment (azayatsfv) [reply APIs/client.py]: changed to los_configuration elsewhere

**Review by darkhipenko — APPROVED** [review]
Approved

**Pull request merged**